services/supervisor/tests/supervisor/service_lifecycle.rs

```diff
@@ -178,3 +177,128 @@ fn stop_already_stopped_service() {
             )
     )
 }
+
+#[test]
+fn resume_stopped_service() {
```

Comment:
Nit: Could you add a test to check that it's impossible to request resume and some other operation regarding the service (e.g., changing its config) in the same update?